src/components/Select.js

```diff
@@ -196,15 +196,13 @@ class Select extends React.Component {
 
     return (
       <div className='mx-select' style={Object.assign({}, this.props.style, { position: 'relative' })}>
-        <div className='mx-select-custom'
+        <button className='mx-select-custom'
           onClick={haltEvent(this._open)}
-          onKeyDown={this._handleKeyDown}
```

Comment:
It looks like `_handleKeyDown` was checking for the `esc` key to close the select.  We might still have to provide a key event handler to preserve that functionality.  We can also remove the `_handleKeyDown` function as it wasn't being used anywhere else.

Reply:
hmmmm. Good catch. Maybe I just go through the effort of making this div work with roles and tabIndex.

Reply:
@cerinman any downsides to just adding the keydown back to the button?

Reply:
nope, it is a valid thing to do if you have to handle something outside of matching the click handler.